sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractFilterForm.java

```diff
@@ -167,7 +167,7 @@ private void onFieldValueChange(String propertyId, Property.ValueChangeEvent eve
 				this.getFieldGroup().commit();
 				this.fireValueChange(true);
 			} catch (FieldGroup.CommitException ex) {
-				ex.printStackTrace();
+				// do nothing
```

Comment:
Sorry, I've just seen that the printStackTrace is already on the development branch, i.e. it's intended. I don't know why I thought that you added it. You can revert this change.